runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -184,6 +200,15 @@ private void appendCheckpointOnSnapshot(Token snapshotTimestamp) {
         } finally {
             rt.getObjectsView().TXEnd();
         }
+
+        return snapshotTimestamp;
+    }
+
+    private Token forceHole() {
+        Token writeToken = rt.getSequencerView().next(streamId).getToken();
+        LogData hole = LogData.getHole(writeToken);
```

Comment:
Its better to use `LogData::useToken` instead of `LogData.getHole`

Reply:
Done.